app/scripts/templates/welcome.html

```diff
@@ -0,0 +1,17 @@
+<h1>Welcome y'all</h1>
+
+{{#supportedBrowser}}
+<div id="install-container">
+  <p>Some details about the add-on</p>
+  <button class="install">Install the add-on</button>
+</div>
+<div id="continue-container" style="display: none;">
```

Comment:
Should we inline styles here, or use some `class="hidden"`?

We have this in the /public/styles/compiled.css:
```css
.is-hidden {
  display: none; }
```